language/testing-infra/functional-tests/src/common.rs

```diff
@@ -95,10 +95,7 @@ impl<T> AsRef<T> for LineSp<T> {
 
 /// Checks if `s` starts with `prefix`. If yes, returns a reference to the remaining part
 /// with the prefix stripped away.
+/// please just use: `s.strip_prefix(prefix)`
 pub fn strip<'a>(s: &'a str, prefix: &str) -> Option<&'a str> {
-    if s.starts_with(prefix) {
-        Some(&s[prefix.len()..])
-    } else {
-        None
-    }
+    s.strip_prefix(prefix)
```

Comment:
Can we just replace all instances of `strip` with `strip_prefix`?

Reply:
Done.